### DISCUSSION TITLE
Add Node Version to Deploy from Github Button

### DISCUSSION THREAD

**Discussion by Smartoneinok** [vercel/community#6635] [Help]
### Summary

I'm looking to see if there is a way to add the node version (example 18.x) to the deploy from github button. The documentation doesn't mention if there is a way to specify the node version you want to use. I tired using the params I thought it could be, but none worked. By default, it is using 20.x... but my project has dependancies that break with that... so its require the original deployment to fail... and then a user has to go in manually and reset the project node version and redeploy... 

Am I missing something? 

### Steps to Reproduce

Deploy this repo... https://github.com/box-community/box-custom-portal-demo/tree/main

You will see the deployment fails... change node version to 18.x in main settings and reploy.

**Comment by amyegan** [comment]
Hey @Smartoneinok. In general, you can set a major Node.js version as the [`engines.node`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines) value in `package.json` to override the version set in Project Settings. That will signal to all environments which Node.js version is needed, not just on Vercel.

https://vercel.com/docs/functions/runtimes/node-js#version-overrides-in-package.json
